tests/run-make/fat-then-thin-lto/lib.rs

```diff
@@ -0,0 +1,8 @@
+#![feature(no_core, lang_items)]
+#![no_core]
```

Comment:
@jieyouxu Do you think this test should use minicore?

Reply:
*Maybe*? As in, `run-make` tests don't get native `//@ add-core-stubs` / `minicore` support by default (not implemented initially).

Reply:
Considering your response and the state of this PR, I'll let someone else clean this up later if they want.

Reply:
Yeah. I didn't add `minicore` support for `run-make` when I implemented minicore initially because it's not always clear *what* about `minicore` run-make tests may want.